apps/pie-docs/src/components/assistive-text/overview/overview.md

```diff
@@ -0,0 +1,214 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: 'Assistive Text'
+    order: 1
+shouldShowContents: true
+---
+
+## Overview
+
+Assistive text helps users understand what kind of information they should enter, how to format it, or any specific requirements related to the input. It can provide examples, suggestions, error messages, validation rules, or any other relevant details that aid in accurate and meaningful input.
+
+The purpose of assistive text is to enhance the user experience by reducing confusion, improving input accuracy, and ensuring users provide the expected information in the desired format.
+
+
+{% contentPageImage {
+    src:"../../../assets/img/components/assistive-text/overview.svg",
```

Comment:
I think this image was added in a previous PR but I think it would be good if we could make the assistive text a bit less vague; something like "Please enter your email address".

Reply:
@RLumleyJET

Reply:
This image was changed.